ipatests/test_integration/test_epn.py

```diff
@@ -15,6 +15,9 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+# W0128 trips on "unused" even if "dummy-variables-rgx=(_.+|unused)"
+# pylint: disable=W0128
+
```

Comment:
pylint complains when there is a double-assignment to unused in the same line. You can remove this disable line and replace `(stdout_text, unused, unused)` with `(stdout_text, unused, _unused)`.

Reply:
Thanks for the hint. Somehow I find that really messy here, but I've changed to that.